### DISCUSSION TITLE
Adding timestamp entities to tiles

### DISCUSSION THREAD

**Discussion by x3style** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#425] [Q&A]
Hi,

I'm trying to add new tiles to the card but timestamp entities do not display correctly.

It should show up like this:
![image](https://user-images.githubusercontent.com/84071923/176288137-85e62f4c-f038-49c1-8c4a-bef77f203f0d.png)

But it shows like this:
![image](https://user-images.githubusercontent.com/84071923/176288043-d7c1a97a-abdd-4d20-9697-d832c2f102eb.png)

Any ideas on the secret sauce to make it work?

**Comment by PiotrMachowski** [comment]
Unfortunately it's not possible currently. You have to create a template sensor with formatted text
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/275
